src/components/JaegerIntegration/RouteHelper.ts

```diff
@@ -37,112 +22,33 @@ const convTagsLogfmt = (tags: string) => {
   const data = logfmtParser.parse(tags);
   Object.keys(data).forEach(key => {
     const value = data[key];
-    // make sure all values are strings
-    // https://github.com/jaegertracing/jaeger/issues/550#issuecomment-352850811
     if (typeof value !== 'string') {
       data[key] = String(value);
     }
   });
   return JSON.stringify(data);
 };
 
-export const logfmtTagsConv = (tags: string | undefined) => {
-  if (!tags) {
-    return null;
-  }
-  let resultTags = '';
-  const jsonTags = JSON.parse(tags);
-  Object.keys(jsonTags).forEach(key => {
-    resultTags += `${key}=${String(jsonTags[key])} `;
-  });
-  return resultTags;
-};
-
-export const getUnixTimeStampInMSFromForm = (
-  startDate: string,
-  startDateTime: string,
-  endDate: string,
-  endDateTime: string
-) => {
-  const start = `${startDate} ${startDateTime}`;
-  const end = `${endDate} ${endDateTime}`;
-  return {
-    start: `${moment(start, 'YYYY-MM-DD HH:mm')
-      .utc()
-      .valueOf()}000`,
-    end: `${moment(end, 'YYYY-MM-DD HH:mm')
-      .utc()
-      .valueOf()}000`
+export const getQueryJaeger = () => {
+  let params: any = {
```

Comment:
When fetching the spans I created the `TracingQuery` type (in types/Tracing.ts), thinking it could be used also for traces queries. I guess it could be reused here (you can extend it with whatever is needed, like limit / tags / etc)

Reply:
I use some transforms and conversions from jaeger Types this is why I created them in types/JaegerInfo